assets/section-featured-product.css

```diff
@@ -53,6 +48,10 @@
   .background-secondary .featured-product {
     padding: 5rem;
   }
+
+  .product--right .product__media-wrapper {
+    order: 2;
+  }
```

Comment:
Yes! Accessibility wise we wouldn't want to do that 👍

Reply:
Ahh great callout, I'll dig into this a bit more.

Reply:
Okay! I caught up with @svinkle (our internal a11y legend) about this. He called out that while it [definitely would be preferable](https://www.w3.org/WAI/WCAG22/Understanding/focus-order.html) to have the visual order match the tab order, this likely wouldn't be a high-SEV issue and that the perf benefit may be a worthwhile tradeoff. We narrowed down on two approaches for when the image gallery is positioned on the right:

- **Skiplink**: Continue using `order` to position and add a product info skiplink to the image gallery. Buyers landing on the page on desktop would tab first to the image gallery skiplink and (optionally) then to the product info, on mobile the behavior is unchanged from how it works today. [Demo](https://screenshot.click/13-47-9hbn5-mzvr8.mp4)
- **Reposition content with JS**: This would be a lot easier with a reactive UI framework but [it's doable](https://gist.github.com/lhoffbeck/4f0dedb9bfc53aba7a547a737fecbcf2). This approach works but is brittle because it relies on the structure of the DOM. [Demo](https://screenshot.click/13-48-7aeps-caucz.mp4)

I put together a test store for each approach and Scott floated them past the Head of Accessibility Innovation at Fable:

> She agreed that it’s a tough spot. In one hand, tab order is not being followed, but given the context that the user is in (navigating content related to the PDP context) and the close proximity of the content, she agreed that the skip link approach would be a viable option. She was sympathetic to the dev experience and worried about the other option could result in brittle code that could break at some point in the future without notice.
Let’s move ahead with the skip link option which should make things easier for you, result in better code, a more performant user experience, and a pretty good accessibility experience.